tests/py2puml/test_py2puml.py

```diff
@@ -1,11 +1,14 @@
 import subprocess
 
-from py2puml import __version__, py2puml
+from py2puml import __version__, __description__, py2puml
 
 
 def test_version():
     assert __version__ == '0.4.0'
 
+def test_description():
+    assert __description__ == 'Generate Plantuml diagrams to document your python code'
+
```

Comment:
IMO, this should be functional tests on the usage of the CLI and/or the script rather than a simple assertion of the values of those.

What do you think?

Reply:
I left like that (in another test file, though), until a more functional test is written. I don't feel the urge for it and I don't want to delay the merging of your contribution